site/ru/beta/migration_guide.ipynb

```diff
@@ -0,0 +1,1621 @@
+{
```

Comment:
stage: https://colab.research.google.com/github/stabuev/docs/blob/master/site/ru/beta/migration_guide.ipynb

Getting a JSON parse error: Unexpected token / in JSON at position 22156

Reply:
Fixed this